Christmas_Lights_Kata/java/src/main/java/org/example/LightService.java

```diff
@@ -0,0 +1,9 @@
+package org.example;
+
+public interface LightService {
+
+    void on(int startRow, int endRow, int startColumn, int endColumn);
+    void off(int startRow, int endRow, int startColumn, int endColumn);
+    void toggle(int startRow, int endRow, int startColumn, int endColumn);
+    long count();
```

Comment:
This violates interface seggregation principle... create seperate interfaces for each